src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using CppSharp.AST;
+using CppSharp.Generators.CSharp;
+
+namespace CppSharp.Passes
+{
+    public class CleanCommentsPass : TranslationUnitPass,ICommentVisitor<bool>
+    {
+        public bool VisitBlockCommand(BlockCommandComment comment)
+        {
+            return true;
+        }
+
+        public override bool VisitDeclaration(Declaration decl)
+        {
```

Comment:
@ddobrev While adding if (base.VisitDeclaration(decl)) return false; i wasn't able to enter the Visit. So i just skipped. If you want to add something else please let me know.